react/features/base/tracks/functions.js

```diff
@@ -105,7 +105,26 @@ export function getLocalAudioTrack(tracks) {
  * @returns {(Track|undefined)}
  */
 export function getLocalTrack(tracks, mediaType) {
-    return tracks.find(t => t.local && t.mediaType === mediaType);
+    return getLocalTracks(tracks).find(t => t.mediaType === mediaType);
```

Comment:
aren't we changing the return type of this function here?

Reply:
no, getLocalTracks returns a Track (it uses filter not map)